StoreMessageContentActivity/__tests__/handler.test.ts

```diff
@@ -253,11 +296,1075 @@ describe("getStoreMessageContentActivityHandler", () => {
+      profileModelMock as any,
+      messageModelMock as any,
+      {} as any,
+      mockServicesPreferencesModel as any,
+      aPastOptOutEmailSwitchDate
+    );
+
+    const result: StoreMessageContentActivityResult = await storeMessageContentActivityHandler(
+      mockContext,
+      {
+        ...aCreatedMessageEvent,
+        message: {
+          ...aNewMessageWithoutContent,
+          senderServiceId: "01234567890" as ServiceId
+        }
+      }
+    );
+
+    expect(result.kind).toBe("FAILURE");
+    if (result.kind === "FAILURE") {
+      expect(result.reason).toEqual("SENDER_BLOCKED");
+    }
+
+    // findServicePreferenceMock is not mocked because it should not be called
+    // senderServiceId: "01234567890" is in blockedInboxOrChannels
+    // => FAILURE
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalled();
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalledTimes(1);
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalledWith([
+      aRetrievedProfile.fiscalCode
+    ]);
+    expect(findServicePreferenceMock).not.toHaveBeenCalled();
+  });
+
+  it("should return a success if message sender service does not exists in profile.blockedInboxOrChannels (LEGACY SETTINGS)", async () => {
+    findLastProfileVersionByModelIdMock.mockImplementationOnce(() =>
+      taskEither.of(
+        some({
+          ...aRetrievedProfile,
+          servicePreferencesSettings: legacyProfileServicePreferencesSettings,
+          blockedInboxOrChannels: {
+            anotherService: [BlockedInboxOrChannelEnum.INBOX]
+          }
+        })
+      )
+    );
+
+    const storeMessageContentActivityHandler = getStoreMessageContentActivityHandler(
+      profileModelMock as any,
+      messageModelMock as any,
+      {} as any,
+      mockServicesPreferencesModel as any,
+      aPastOptOutEmailSwitchDate
+    );
+
+    const result: StoreMessageContentActivityResult = await storeMessageContentActivityHandler(
+      mockContext,
+      {
+        ...aCreatedMessageEvent,
+        message: {
+          ...aNewMessageWithoutContent,
+          senderServiceId: "01234567890" as ServiceId
+        }
+      }
+    );
+
+    expect(result.kind).toBe("SUCCESS");
+    if (result.kind === "SUCCESS") {
+      expect(result.blockedInboxOrChannels).toEqual([]);
+    }
+
+    // findServicePreferenceMock is not mocked because it should not be called
+    // senderServiceId: "01234567890" is not in blockedInboxOrChannels
+    // => SUCCESS
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalled();
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalledTimes(1);
+    expect(findLastProfileVersionByModelIdMock).toHaveBeenCalledWith([
+      aRetrievedProfile.fiscalCode
+    ]);
+    expect(findServicePreferenceMock).not.toHaveBeenCalled();
+  });
+
+  it("should return a success if message sender service exists in profile.blockedInboxOrChannels but only blocks EMAIL (LEGACY SETTINGS)", async () => {
```

Comment:
ditto

Reply:
Removed in https://github.com/pagopa/io-functions-services/pull/142/commits/b9d58101e54264cc46f6fa6221efdc7febeba3e8